src/inferenceql/query/base.cljc

```diff
@@ -22,14 +22,14 @@
   - parse - a parsing function"
   [s db parse]
   (let [node-or-failure (parse s)]
+    (tap> #:base.query{:node node-or-failure})
```

Comment:
interesting, do you recommend leaving this in for production code? I like the idea of more logging via `tap>`, I just don't know good practices yet.

Reply:
Zane requested this. The nice thing about `tap>` is it's very lightweight if no taps are added.

The less nice thing about `tap>` is well, everything else, IMO. :P I've tried REBL and Portal, and have yet to find a tap/datafy visualizer I actually think is worth using over logs. They're just so slow to navigate in.

For now, I've taken to running:
```clojure
(do
  (when (ns-resolve 'user 'log-tap)
    (remove-tap user/log-tap))
  (intern 'user 'log-tap #(do (prn %) (println)))
  (add-tap user/log-tap))
```

so I get some semblance of logs.

Reply:
Nothing now... in Emmy I have used Timbre but I'm not doing anything special to consume the logs.

Reply:
FWIW, I'm a fan of mulog, and other structured logging libs like pedestal.log. Timbre's not structured, iiuc, but it's already cross-platform, and it eliminates much of the need to fiddle with Java logging backend config files, which are both major pluses.

I think it's fine to leave the tap>'s in, but I might add a logging helper for it.

Reply:
>We can't choose a solution when we haven't articulated the problem.

For me, I find it help for debugging, since I can see all output at once, and search the REPL if necessary. Cloud servers are a future problem, imo. And I feel I don't know enough about third-party users yet to offer an opinion.